lib/buildsystems/cmake.rb

```diff
@@ -1,13 +1,13 @@
 require 'package'
 
 class CMake < Package
-  property :cmake_options
+  property :cmake_options, :pre_cmake_options
 
   def self.build
-    puts "Additional cmake_options being used: #{@cmake_options.nil? || @cmake_options.empty? ? '<no cmake_options>' : @cmake_options}".orange
+    puts "Additional cmake_options being used: #{@pre_cmake_options.nil? || @pre_cmake_options.empty? ? '<no pre_cmake_options>' : @pre_cmake_options} #{@cmake_options.nil? || @cmake_options.empty? ? '<no cmake_options>' : @cmake_options}".orange
```

Comment:
The `pre_cmake_options` will not be empty? (no need to check if it is an empty string)

Reply:
This should be the same for all of the `pre_` and non-`pre_` options in all of the buildsystems files.

Do you have suggested code for all of it?

Reply:
I will try removing `@cmake_options.empty?` and so on to see if anything breaks. :)

Reply:
Changes made!